### PR TITLE
Matrix operations code added

### PR DESCRIPTION
**Pull request by drishika2002**
Code added for the issue: ( Matrices operations #727 )
Please review @OjusWiZard @rudrakshi99 @tarun26091999

### CODE REVIEW

File: Code/C++/matrix_operations.cpp [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+		// Formula for finding inverse of a matrix: A^(-1) = adj(A) / det(A)
+		cout<<"\nINVERSE OF MATRIX A: "<<endl;
+		inverse_of_matrices(rows,a);
+        }
+	return 0;
+}
+
+/*
+
+OUTPUT:
```
Comment (Rohan27s) [inline Code/C++/matrix_operations.cpp:181]: Give 2 Test Cases

File: Code/C++/matrix_operations.cpp [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+2       8       14
+
+INVERSE OF MATRIX A:
+5       -4      2
+-10     8       -3
+4       -3      1
+
+Time Complexity: O(n^2)
+Space Complexity: O(n^2)
+
```
Comment (Rohan27s) [inline Code/C++/matrix_operations.cpp:210]: remove this space

File: Code/C++/matrix_operations.cpp [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+		cout<<"\nMULTIPLICATION OF TWO MATRICES: "<<endl;
+		product_of_matrices(rows,cols,a,b);
+
+		// Formula for finding inverse of a matrix: A^(-1) = adj(A) / det(A)
+		cout<<"\nINVERSE OF MATRIX A: "<<endl;
+		inverse_of_matrices(rows,a);
+        }
+	return 0;
+}
+
```
Comment (Rohan27s) [inline Code/C++/matrix_operations.cpp:178]: remove unnecessary spaces

File: Code/C++/matrix_operations.cpp [hunk 4]
```diff
@@ -0,0 +1,211 @@
+/* C++ CODE TO SHOW DIFFERENT OPERATIONS ON 2D ARRAY LIKE SUBTRACTION, ADDITION, MULTIPLICATION AND FINDING THE INVERSE OF A MATRIX */
+
```
Comment (Rohan27s) [inline Code/C++/matrix_operations.cpp:2]: Give a description of your code

**Review by Rohan27s — CHANGES_REQUESTED** [review]

File: Code/C++/matrix_operations.cpp (hunk 2 above)
Comment (drishika2002) [reply Code/C++/matrix_operations.cpp]: Done sir

File: Code/C++/matrix_operations.cpp (hunk 1 above)
Comment (drishika2002) [reply Code/C++/matrix_operations.cpp]: Done sir

File: Code/C++/matrix_operations.cpp (hunk 4 above)
Comment (drishika2002) [reply Code/C++/matrix_operations.cpp:8]: Added sir, please check sir if it's ok now.

File: Code/C++/matrix_operations.cpp (hunk 3 above)
Comment (drishika2002) [reply Code/C++/matrix_operations.cpp:186]: Done sir, please check once

**Review by Rohan27s — APPROVED** [review]

**Pull request merged**